### PR TITLE
breoganpardo

### PR DESCRIPTION
**Pull request by breoganpardo**
Hopefully this is the right one

### CODE REVIEW

**Comment by daguito81** [comment]
@breoganpardo  Hey Breo, you still have that fix.exe file. Are you planning on leaving that in the root directory?

File: fix.exe [hunk 1]
```diff
@@ -0,0 +1 @@
+Executable
```
Comment (astrojuanlu) [inline fix.exe]: Let's remove this file

**Comment by astrojuanlu** [comment]
@breoganpardo This one is correct! One file to go and we will merge.

**Review by astrojuanlu — APPROVED** [review]
:top:

**Comment by astrojuanlu** [comment]
:warning: Next steps :warning: 

Your pull request has been merged to `master`. That means that, if you don't do anything, your local `master` and `breoganpardo/master` are outdated with respect to `origin/master`. On the other hand, you should avoid working on this branch anymore in the future.

To put yourself in a clean state again, you have to:

1. Click "remove branch" here in this pull request (don't click "remove fork"!)
2. `git checkout master`
3. `git fetch origin` (assuming https://github.com/Juanlu001/ie-mbd-advanced-python/ is `origin` for you)
4. `git merge --ff-only origin master` (this will update your local `master` with `origin/master`, and fail if you accidentally made any commit in `master`)
5. `git fetch -p breoganpardo` (assuming https://github.com/breoganpardo/ie-mbd-advanced-python/ is `breoganpardo` for you) :star2: this will acknowledge the removal of the branch :star2:
6. `git push breoganpardo master` (this will update your fork with respect to `origin`)

The objective of this process, **which has to be repeated for every pull request**, is to be able to submit a second one without git problems.

~ If in doubt, please ask ~

:rainbow:

**Pull request merged**